download-clis.sh

```diff
@@ -13,4 +13,7 @@ chmod 755 ./clis-unpacked/kubectl
 mv ./clis-unpacked/oc /usr/local/bin/oc
 mv ./clis-unpacked/kubectl /usr/local/bin/kubectl
 
+# Install htpasswd utility
+apt-get update && apt-get install -y apache2-utils
+
 echo 'oc and kubectl cli install completed.'
```

Comment:
Minor suggestion to help us debug problems later.
- Rename the file to something more generic like `install-dependencies.sh`
- Update the echo message to include htpasswd.

Reply:
Renamed the file and updated the echo messages.